searchlib/src/vespa/searchlib/expression/debugwaitfunctionnode.cpp

```diff
@@ -28,22 +25,13 @@ DebugWaitFunctionNode::DebugWaitFunctionNode(ExpressionNode::UP arg, double wait
 {
 }
 
+using std::chrono::microseconds;
+
 bool
 DebugWaitFunctionNode::onExecute() const
 {
-    FastOS_Time time;
-    time.SetNow();
-    double millis = _waitTime * 1000.0;
+    fastos::StopWatch::waitAtLeast(microseconds(static_cast<long>(_waitTime * 1000000)), _busyWait);
```

Comment:
I prefer long(expr) to static cast in these cases

Reply:
Fixed